pkgs/os-specific/darwin/raycast/default.nix

```diff
@@ -50,7 +50,7 @@ stdenvNoCC.mkDerivation (finalAttrs: {
   meta = {
     description = "Control your tools with a few keystrokes";
     homepage = "https://raycast.app/";
-    license = with lib.licenses; [ unfree ];
+    license = lib.licenses.free;
```

Comment:
This seems wrong, this is definitely unfree?

Reply:
Sorry that was oversight on my part!